docs/source/pkg.rst

```diff
@@ -38,7 +38,7 @@ An example call to install a few packages on 2 hosts (with hostnames like
 
 .. _pkg-remove:
 
-``--remove``
+--remove
```

Comment:
I could easily pull the backticks and smartypants change (they are intertwined) into a second PR to be debated.  :)